data/mods/ssb/moves.js

```diff
@@ -3352,16 +3347,7 @@ let BattleMovedex = {
 		flags: {},
 		terrain: 'nightmarefield',
```

Comment:
pseudoWeather over terrain imo

Reply:
Done.

Reply:
Wait. Do not merge. Nightmare effect not activating.

Reply:
My b, add this to nightmare configuration

```suggestion
		onAfterMoveSecondarySelf: function () {
		this.addPseudoWeather('nightmarefield');
		},
```

Reply:
Make sure its properly tabbed ofc